src/scaffoldmaker/meshtypes/meshtype_3d_wholebody1.py

```diff
@@ -0,0 +1,435 @@
+"""
+Generates body coordinates using a solid cylinder of all cube elements,
+ with variable numbers of elements in major, minor, shell and for each section of abdomen, thorax, neck and head.
+"""
+
+from __future__ import division
+import math
+import copy
+from opencmiss.utils.zinc.field import findOrCreateFieldCoordinates, findOrCreateFieldGroup,\
+    findOrCreateFieldStoredString, findOrCreateFieldStoredMeshLocation, findOrCreateFieldNodeGroup
+from scaffoldmaker.meshtypes.scaffold_base import Scaffold_base
+from scaffoldmaker.utils.meshrefinement import MeshRefinement
+from scaffoldmaker.utils.cylindermesh import CylinderMesh, CylinderShape, CylinderEnds, CylinderCentralPath
+from scaffoldmaker.utils.zinc_utils import exnodeStringFromNodeValues
+from scaffoldmaker.scaffoldpackage import ScaffoldPackage
+from scaffoldmaker.meshtypes.meshtype_1d_path1 import MeshType_1d_path1
+from opencmiss.zinc.node import Node
+from opencmiss.zinc.element import Element
+from scaffoldmaker.annotation.annotationgroup import AnnotationGroup, findOrCreateAnnotationGroupForTerm, getAnnotationGroupForTerm, mergeAnnotationGroups
+from scaffoldmaker.annotation.body_terms import get_body_term
+from opencmiss.utils.zinc.general import ChangeManager
+from opencmiss.zinc.field import Field
+from opencmiss.utils.zinc.finiteelement import get_element_node_identifiers
+from scaffoldmaker.utils.eft_utils import remapEftNodeValueLabelsVersion
+
+
+class MeshType_3d_wholebody1(Scaffold_base):
+    """
+Generates body coordinates using a solid cylinder of all cube elements,
+ with variable numbers of elements in major, minor, shell and for each section of abdomen, thorax, neck and head.
+    """
+    cylinder1Settings = {
+                'Coordinate dimensions': 3,
+                'D2 derivatives': True,
+                'D3 derivatives': True,
+                'Length': 3.5,
+                'Number of elements': 1
+            }
+
+    centralPathDefaultScaffoldPackages = {
+        'Cylinder 1': ScaffoldPackage(MeshType_1d_path1, {
```

Comment:
'Cylinder 1' is not a good name. Change to 'Default'? I initially wasn't keen on exposing this, but we could maintain some pre-fit paths for different species in future...

Reply:
Done